router/src/server.rs

```diff
@@ -927,15 +1006,17 @@ pub async fn run(
         docker_label: option_env!("DOCKER_LABEL"),
     };
 
-    MODEL_ID.set(model_id.clone()).unwrap_or_else(|_| {
-        panic!("MODEL_ID was already set!");
-    });
     DEFAULT_ADAPTER_SOURCE
         .set(adapter_source.clone())
         .unwrap_or_else(|_| {
             panic!("DEFAULT_ADAPTER_SOURCE was already set!");
         });
 
+    // Kick off thread here that writes to the log file
+    let (tx, rx) = mpsc::channel(32);
+    tokio::spawn(request_logger(rx));
```

Comment:
Should this be conditioned on `REQUEST_LOGGER_URL` being set?

Reply:
Makes sense!